src/lib/staged_ledger/staged_ledger.mli

```diff
@@ -343,9 +351,12 @@ val all_work_pairs :
 (** Statements of all the pending work in t*)
 val all_work_statements_exn : t -> Transaction_snark_work.Statement.t list
 
+val dummy_transaction_pool_proxy : transaction_pool_proxy
```

Comment:
Maybe we could remove it from `.mli` file, as it is not used anywhere outside of corresponding `.ml` file?

Reply:
I think this is not true